browser/ui/brave_browser.cc

```diff
@@ -163,6 +163,17 @@ void BraveBrowser::ResetTryToCloseWindow() {
   Browser::ResetTryToCloseWindow();
 }
 
+void BraveBrowser::UpdateTargetURL(content::WebContents* source,
+                                   const GURL& url) {
+  GURL target_url = url;
+  if (url.SchemeIs(content::kChromeUIScheme)) {
+    GURL::Replacements replacements;
+    replacements.SetSchemeStr(content::kBraveUIScheme);
+    target_url = target_url.ReplaceComponents(replacements);
+  }
```

Comment:
Could it be worth replacing 'chrome-untrusted' too?

Reply:
I'm not so sure about that because chrome-untrusted is never meant to be displayed to users